components/ordered-list/src/index.js

```diff
@@ -0,0 +1,35 @@
+// https://govuk-static.herokuapp.com/component-guide/government_navigation
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import glamorous from 'glamorous';
+
+const GOrderedListInner = glamorous.ol(
+  {
+    boxSizing: 'border-box',
```

Comment:
i think it may be better to put in font-related style definitions here

this is how govuk-frontend does things, and the other older govuk styling too.

arguably the `ListItem` component shouldn't need much styling - govuk-frontend just has margin-bottom applied to `li` items (with media queries)

Reply:
this comment of course also applies to the `unordered-list` component too 😀